modules/zypper/README.md

```diff
@@ -0,0 +1,31 @@
+Zypper
+===
+
+Defines [zypper][1] aliases.
+
+Aliases
+-------
+
+  - `zypc`  # Cleans the cache.
```

Comment:
Might want to add `zyp` as a shorthand for `zypper` since that would go along with the rest of the aliases.